services/121-service/src/registration/const/filter-operation.const.ts

```diff
@@ -13,7 +13,7 @@ export const AllowedFilterOperatorsNumber = [
   FilterOperator.NULL,
 ];
 
-const dataSearchableColumn = 'data.(value)';
+const dataSearchableColumn = 'data.value';
```

Comment:
I made these changes to get the TS compiler to stop complaining, but I do not know if they are valid changes. I see that the previous version of the file was written during this task: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/27569

Maybe @PeterSmallenbroek you can have a look at this?

Reply:
(Note: this is because we are updating to a major version of paginate with breaking changes - maybe it's worth checking the release notes beside this change that I've made).